db/schema.rb

```diff
@@ -31,12 +31,6 @@
   add_index "applicants", ["application_id"], name: "index_applicants_on_application_id", using: :btree
 
   create_table "applications", force: :cascade do |t|
-    t.string   "title"
-    t.string   "first_name"
-    t.string   "last_name"
-    t.date     "date_of_birth"
-    t.string   "ni_number"
-    t.boolean  "married"
     t.string   "status"
```

Comment:
I've just noticed that these field have been deleted from my schema, despite me not touching them... I'm going to update the file and force push this branch.